TrackletAlgorithm/TrackletProcessor.h

```diff
@@ -297,69 +273,21 @@ inline ap_uint<(1<<(2*kNBitsBuffer))> nearFullTEUnitInit() {
 
 ////////////////////////////////////////////////////////////////////////////////
 
-#include "TrackletCalculator_calculate_LXLY.h"
+//#include "TrackletCalculator_calculate_LXLY.h"
```

Comment:
@aryd Since this file has been removed, let's just remove this line:
```suggestion
```

Reply:
Fixed.